docs/pages/textarea.js

```diff
@@ -53,6 +53,265 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-dont-placeholder"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="Tell us your story"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use helperText to provide additional context that will aid the user in most effectively inputing information."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-helpertext"
+        label="Explain what people can see in this Pin"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        helperText="This text will be read aloud by screen readers"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use placeholder to provide any information necessary to filling out the form field. Placeholder text disappears after the user begins entering data and should not contain crucial information."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-helpertext"
+        label="Campaign description"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="Maximum of 500 characters"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Set the height of TextArea using row to ensure that the typical amount of text entered will be visible without needing to scroll."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-height"
+        label="Have feedback on this product?"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="Tell us about your experience, what you love, or what we could improve."
+        helperText="Please don't submit passwords, email addresses, or other sensitive or personal info."
+        rows={5}
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Set the row prop to less than 2. Use TextField instead, if only a single line of text is expected."
```

Comment:
The second sentence is a little awkward in its current form. Consider switching the clauses to "If only a single line of text is expected, use TextField instead.".

Also: `row` should have backticks.

Reply:
Done - cleaned up the second sentence a little differently, let me know if it makes sense to you.